Resources/Prototypes/RPD/rpd.yml

```diff
@@ -0,0 +1,258 @@
+- type: rcd
+  id: PipeFourway
+  category: Piping
+  sprite: /Textures/Interface/Radial/RPD/fourway.png
+  mode: ConstructObject
+  prototype: GasPipeFourway
+  cost: 0.5
+  delay: 0
+  rules:
+  - MustBuildOnSubfloor
+  rotation: User
+  fx: EffectRCDConstruct0
+
+- type: rcd
+  id: PipeStraight
+  category: Piping
+  sprite: /Textures/Interface/Radial/RPD/straight.png
+  mode: ConstructObject
+  prototype: GasPipeStraight
+  cost: 0.5
+  delay: 0
+  rules:
+  - MustBuildOnSubfloor
+  rotation: User
+  fx: EffectRCDConstruct0
+
+- type: rcd
+  id: PipeBend
+  category: Piping
+  sprite: /Textures/Interface/Radial/RPD/bend.png
+  mode: ConstructObject
+  prototype: GasPipeBend
+  cost: 0.5
+  delay: 0
+  rules:
+  - MustBuildOnSubfloor
+  rotation: User
+  fx: EffectRCDConstruct0
+
+- type: rcd
+  id: PipeTJunction
+  category: Piping
+  sprite: /Textures/Interface/Radial/RPD/tjunction.png
+  mode: ConstructObject
+  prototype: GasPipeTJunction
+  cost: 0.5
+  delay: 0
+  rules:
+  - MustBuildOnSubfloor
+  collisionMask: Impassable
+  rotation: User
+  fx: EffectRCDConstruct0
+
+- type: rcd
+  id: PressurePump
+  category: PumpsValves
+  sprite: /Textures/Interface/Radial/RPD/pump_pressure.png
+  mode: ConstructObject
+  prototype: GasPressurePump
+  cost: 1
+  delay: 0
+  rules:
+  - MustBuildOnSubfloor
+  collisionMask: Impassable
+  rotation: User
+  fx: EffectRCDConstruct0
+
+- type: rcd
+  id: VolumetricPump
+  category: PumpsValves
+  sprite: /Textures/Interface/Radial/RPD/pump_volume.png
+  mode: ConstructObject
+  prototype: GasPressurePump
```

Comment:
This field for the volumetric pump spawns a normal pressure pump instead due to wrong prototype.